pulser/tests/test_register.py

```diff
@@ -62,6 +63,154 @@ def test_creation():
     assert np.all(np.array(reg6._coords) == coords_)
 
 
+def test_rectangle():
+    # Check rows
+    with pytest.raises(ValueError, match="The number of rows"):
+        reg = Register.rectangle(0, 2)
```

Comment:
Since you'll never use `reg`, just do this instead:
```suggestion
       Register.rectangle(0, 2)
```